fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/logical/EliminateGroupByConstant.java

```diff
@@ -50,6 +51,7 @@ public Rule build() {
             List<NamedExpression> outputExprs = aggregate.getOutputExpressions();
             Set<Expression> slotGroupByExprs = Sets.newLinkedHashSet();
             for (Expression expression : groupByExprs) {
+                expression = FoldConstantRule.INSTANCE.rewrite(expression);
```

Comment:
remove constant do not need to do FoldConstantRule, just use Expression#isConstant

Reply:
We could do this after the process for FoldConstant  is complete, for now, there still might be some expression like `1+1` not get folded here